.devcontainer/Dockerfile

```diff
@@ -13,6 +13,9 @@ ENV LC_ALL en_US.UTF-8
 RUN apt-get update
 RUN apt-get install -y locales && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
 RUN apt-get -fy install vim sudo bash-completion build-essential gdb docker.io
+RUN apt-get update && apt-get install -y valgrind gcovr
+RUN pip3 install virtualenv
+RUN pip3 install mbed-tools --upgrade
```

Comment:
How do we get the right version of CMake ?

Reply:
This inherits from Saheer's stable [image](https://github.com/0xc0170/docker-hub/tree/master/mbed-os-env), which has the right CMake version.